app/src/main/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsViewModel.kt

```diff
@@ -115,6 +118,21 @@ class GroupConversationDetailsViewModel @Inject constructor(
         updateGuestRemoteRequest(false)
     }
 
+    fun onServiceDialogDismiss() {
+        updateState(
+            groupOptionsState.copy(
+                loadingServicesOption = false,
+                changeServiceOptionConfirmationRequired = false,
+                isServicesAllowed = !groupOptionsState.isServicesAllowed
+            )
+        )
+    }
+
+    fun onServiceDialogConfirm() {
```

Comment:
```suggestion
    fun onServiceDialogConfirmed() {
```

Reply:
I suggested this name because I assumed that you call it after finishing the change on the api/server. right?